classes/03class/exercises/c03-make01/frdvo/README.md

```diff
@@ -0,0 +1,60 @@
+# C03-MAKE01
+
+## Make
+- [Dockerfile](Dockerfile)
+- [Makefile](Makefile)
+
+## Command Execution Output
+
+- Execution and output of the command `make build`
+```
+$ make build
+Getting the short commit sha and saving on index.html
+Building container...
+docker build --tag frdvo/make01:af435c2 .
+Sending build context to Docker daemon  6.656kB
+Step 1/2 : FROM httpd:2.4.41-alpine
+ ---> 54b0995a6305
+Step 2/2 : COPY index.html /usr/local/apache2/htdocs/
+ ---> Using cache
+ ---> c0bf412fa4b6
+Successfully built c0bf412fa4b6
+Successfully tagged frdvo/make01:af435c2
+Check below your built container
+REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
+frdvo/make01        af435c2             c0bf412fa4b6        42 minutes ago      107MB
+Your container is good to go! make push it!
+Made with WSL2🐧💕🐳
+```
+
+- Execution and output of the command `make push`
+```
+$ make push
+Pushing the container to Docker Hub
+docker push frdvo/make01:af435c2
+The push refers to repository [docker.io/frdvo/make01]
+eef35c691a5e: Layer already exists
+8e24b43b535f: Layer already exists
+1dfb4a5d0ecd: Layer already exists
+3313c0c04ade: Layer already exists
+310889822143: Layer already exists
+beee9f30bc1f: Layer already exists
+af435c2: digest: sha256:1b66aa31e150981422c8ce5c537ae4d7b07fd140248f5ddda7411ed766354db4 size: 1569
+Made with WSL2🐧💕🐳
+```
+
+- Execution and output of the command `make run`
+```
+$ make run p=5000
+Lazy container runner to choose a port use make run p=<Port Number> otherwise it will run on port 80
+docker run --publish 5000:80 frdvo/make01:af435c2
```

Comment:
Can you run more than one container with this exact same command? 

(No, you can't. Ports will conflict in the host and won't let you bind to port 5000 again).

Small fix needed here, as per the exercise description: `The port to be mapped for the container should be dynamically provided during the make run command.`

Check the `-p 80` option.